nautobot/docs/release-notes/version-2.0.md

```diff
@@ -211,3 +211,64 @@ Check out the specific changes documented in the table at [UI and REST API Filte
 Support for RQ and `django-rq`, deprecated since Nautobot 1.1.0, has been fully removed from Nautobot 2.0.
```

Comment:
When do we want to review the Release Overview section for completeness/accuracy?

Reply:
Added as a to-do item for next sprint: https://github.com/orgs/nautobot/projects/4/views/6?filterQuery=sprint%3A%22Sprint+45%22

Reply:
Would be *better* IMO to review this and the upgrading guide before we ship the alpha, but OK.